components/aws/sagemaker/tests/integration_tests/scripts/run_integration_tests

```diff
@@ -17,7 +17,7 @@ REGION=${REGION:-"$(aws configure get region)"} # Deployment region
 
 ### Configuration parameters
 EKS_EXISTING_CLUSTER=${EKS_EXISTING_CLUSTER:-""} # Use an existing EKS cluster
-EKS_CLUSTER_VERSION=${EKS_CLUSTER_VERSION:-"1.18"} # EKS cluster K8s version
+EKS_CLUSTER_VERSION=${EKS_CLUSTER_VERSION:-"1.19"} # EKS cluster K8s version
```

Comment:
1.19 will be outdated soon, can we upgrade it to latest version like 1.20 or even 1.21 ?

Reply:
Yea I was wondering the same so we discussed today and decided to run each version for a few days before jumping to the latest one. That way we know none of them is broken. So let's merge this PR as is and I will take up the task of updating to 1.20 and then to 1.21 in a week's time.